HealthMonitoring.Monitors.Nsb6.RabbitMq/Nsb6Monitor.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using HealthMonitoring.Monitors.Nsb6.Messages;
+using NServiceBus;
+
+namespace HealthMonitoring.Monitors.Nsb6.RabbitMq
+{
+    public class Nsb6Monitor : IHealthMonitor, IDisposable
+    {
+        private static readonly HealthInfo MessageTimeoutResponse = new HealthInfo(HealthStatus.Faulty, new Dictionary<string, string> { { "message", "health check timeout" } });
+        private readonly TimeSpan _messageTimeout;
+        public string Name => "nsb6.rabbitmq";
+        private IEndpointInstance _endpoint;
+
+        public Nsb6Monitor()
+        {
+            _messageTimeout = GetMessageTimeout();
+            var endpointConfiguration = EndpointConfigurationProvider.BuildEndpointConfiguration(_messageTimeout);
+            StartAsync(endpointConfiguration).GetAwaiter().GetResult();
```

Comment:
this looks strange but I think I understand that it is not blocking the ctor for the time being of the endpoint being alive, but just to finish the startup process...